trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-summary-helper.ts

```diff
@@ -12,11 +20,16 @@ export const reconciliationStatusCodeToDisplayStatus: Record<UtilisationReportRe
   RECONCILIATION_COMPLETED: 'Report completed',
 };
 
+const getUtilisationReportDisplayFrequency = (reportPeriod: ReportPeriod): UtilisationReportDisplayFrequency => {
```

Comment:
Functions missing docstring, all the functions should always have a docstring please.